patterns/19.async-nature-of-setState.md

```diff
@@ -30,16 +30,16 @@ Not being in control means it's not able to do any perf optimisations so it's be
 make sure the code that follows has access to the latest information available.
 
 ```javascript
-var TestComponent = React.createClass({
+class TestComponent extends React.Component {
   displayName: 'TestComponent',
```

Comment:
Would it make sense to make it a static getter instead of a property? 

Something like this:

```javascript
class TestComponent extends React.Component {
  static get displayName() {
    return 'TestComponent';
  }
}
```